grammar.py

```diff
@@ -292,7 +292,7 @@ def _generate_code(self, num_lines, initial_variables=[], last_var=0):
                 context = tmp_context
             except RecursionError as e:
                 print('Warning: ' + str(e))
-        for i in range(len(context['lines']) / 100):
+        for i in range(int(len(context['lines']) / 100)):
```

Comment:
You can use the `//` operator to get integer division, if that's what you're doing.